crates/bevy_ecs/src/schedule/system_descriptor.rs

```diff
@@ -134,6 +154,13 @@ pub trait ParallelSystemDescriptorCoercion<Params> {
     /// Specifies that the system is exempt from execution order ambiguity detection
     /// with other systems in this set.
     fn in_ambiguity_set(self, set: impl AmbiguitySetLabel) -> ParallelSystemDescriptor;
+
+    /// Specifies that this system is ambiguous and must be ignored by ambiguity detection.
+    fn silence_ambiguity_checks(self) -> ParallelSystemDescriptor;
```

Comment:
I know this was already renamed, but I think this could be named something like `allow_ambiguity()`. It's not as long (simpler to remember) and it also suggests that other internal systems related to ambiguity should skip this system (not just the logging).

Reply:
Good idea: fixed!